src/boot/categories.toml

```diff
@@ -279,6 +279,33 @@ Crates to help develop software capable of adapting to various \
 languages and regions.\
 """
 
+[language-implementations]
+name = "Language implementations"
+description = """
+Compilers, interpreters, transpilers implemented for particular programming languages.\
+"""
+
+[language-implementations.categories.transpilers]
+name = "Transpiler implementations"
+description = """
+Transpilers deal with source to source translations, such as\
+Python <-> Rust
+"""
+
+[language-implementations.categories.compilers]
+name = "Compiler implementations"
+description = """
+Compilers deal with source to machine translations, like compiling\
+Rust for x86 architectures.
+"""
+
+[language-implementations.categories.interpreters]
+name = "Interpreter implementations"
+description = """
+Interpreters deal with source to some IR translations, such as AST or\
+bytecode based interpreters like CPython or Rust's MIRI.
+"""
+
```

Comment:
```suggestion
```

we discussed this in one of our team meetings and since the lines between these are quite blurry we decided that we'd prefer to only have a single category without subcategories for now.